.github/workflows/main.yaml

```diff
@@ -0,0 +1,19 @@
+on: [push, pull_request]
```

Comment:
I suppose it would be easier to discuss the GitHub workflow in a separate pull request.

Reply:
Should we take this bit out then?